test/SdkNetCoreTests/TestConfig.cs

```diff
@@ -20,46 +21,52 @@ class TestConfig
         public string ReturnUrl { get; set; }
         public string UserId { get; set; }
         public string OAuthBasePath { get; set; }
-        public string PrivateKeyFilename { get; set; }
+        public string PrivateKey { get; set; }
         public int ExpiresInHours { get; set; }
 
         public List<string> EnvelopeIdsList { get; set; }
 
         public string IntegratorKeyNoConsent { get; set; }
 
-        public string PrivateKeyNoConsentFilename { get; set; }
+        public string PrivateKeyNoConsent { get; set; }
 
         public string BrandLogo { get; set; }
 
         public string BrandId { get; set; }
 
         public TestConfig(string username = null, string password = null, string integratorKey = null, string host = null, string recipientEmail = null, string recipientName = null, string templateRoleName = null, string templateId = null, string returnUrl = null)
         {
+            string userIdFromEnv = Environment.GetEnvironmentVariable("userid");
+            string integratorKeyFromEnv = Environment.GetEnvironmentVariable("integratorkey");
+            string templateIdFromEnv = Environment.GetEnvironmentVariable("templateid");
+            string integratorKeyNoConsentFromEnv = Environment.GetEnvironmentVariable("integratorkeynoconsent");
+            string brandIdfromEnv = Environment.GetEnvironmentVariable("brandid");
+
             this.Host = (host != null) ? host : "https://demo.docusign.net/restapi";
-            this.Username = (username != null) ? username : "node_sdk@mailinator.com";
-            this.Password = (password != null) ? password : "{PASSWORD}";
```

Comment:
Remove all references to password. We don't need it anymore.

Reply:
removed!